program.c

```diff
@@ -219,6 +219,23 @@ int erase_execute(struct qdl_device *qdl, int (*apply)(struct qdl_device *qdl, s
 	return 0;
 }
 
+static struct program *program_find_partition(const char *partition)
```

Comment:
Please write somewhat sensible commit message, descring why do you need to perform these changes and/or what kind of refactoring was performed

Reply:
Extended commit message. Let me know if you have any additional suggestions or objections. Thanks!